--- Configure.org	Sat May  1 19:59:05 1999
+++ Configure.help	Sat May  1 20:08:56 1999
@@ -4541,6 +4541,20 @@
   driver is the only voice-supporting drivers. See
   Documentation/isdn/README.audio for more information.
 
+ISDN diversion services support
+CONFIG_ISDN_DIVERSION
+  This option allows you to use some supplementary diversion
+  services in conjunction with the HiSax driver on an EURO/DSS1
+  line. Supported options are CD (call deflection), CFU (Call 
+  forward unconditional), CFB (Call forward when busy) and CFNR 
+  (call forward not reachable).
+  Additionally the actual CFU, CFB and CFNR state may be 
+  interrogated. The use of CFU, CFB, CFNR and interrogation may
+  be limited to some countries. The keypad protocol is still not
+  implemented.
+  CD should work in all countries if this service has been sub-
+  scribed.
+
+HiSax driver for HFC-S+, HFC-SP and HFC-PCMCIA cards
+CONFIG_HISAX_HFC_SX
+  This option allows you to add support for for ISDN-cards supplied
+  with CCDs HFC-S+, HFC-SP and HFC-PCMCIA-cards with the SP chip.
+  Drivers for the very old HFC-S chip and the new busmaster HFC-PCI
+  are separate selectable drivers and not covered by this option.
+  Due to problems with cards only equipped with 8K of RAM this driver
+  only supports cards equipped with 32K RAM like CCD recommends.
+
 ICN 2B and 4B support
 CONFIG_ISDN_DRV_ICN
   This enables support for two kinds of ISDN-cards made by a German
